src/components/pages/PatientsFullDetailsSearch/PatientsFullDetailsSearch.js

```diff
@@ -11,9 +11,16 @@ import { fetchPatientCountsRequest } from '../../../ducks/fetch-patient-counts.d
 import { fetchPatientsCountsOnMountAndUpdate } from '../../../utils/HOCs/fetch-patients.utils';
 import { fetchPatientOnSearch } from '../../../utils/HOCs/fetch-patient-on-search.utils';
 import { fetchAdvancedPatientSearchRequest } from '../../../ducks/fetch-advanced-patient-search.duck';
+import { setCurrentPageOffsetStart } from '../../../ducks/set-current-page-offset.duck'
 
-const mapDispatchToProps = dispatch => ({ actions: bindActionCreators({ fetchBasicPatientSearchRequest, fetchAdvancedPatientSearchRequest, fetchPatientCountsRequest }, dispatch) });
-
+const mapDispatchToProps = dispatch => (
+  { actions: bindActionCreators({
+                fetchBasicPatientSearchRequest,
+                fetchAdvancedPatientSearchRequest,
+                fetchPatientCountsRequest,
+                setCurrentPageOffsetStart },
+              dispatch)
```

Comment:
@olegmdev please look at the spacing and make corresponding changes, this does not look right